components/tabs/tab-mixin.js

```diff
@@ -101,16 +101,15 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 
 	update(changedProperties) {
 		super.update(changedProperties);
-		changedProperties.forEach((oldVal, prop) => {
-			if (prop === 'selected') {
-				this.ariaSelected = this.selected;
-				if (this.selected === 'true') {
-					this.dispatchEvent(new CustomEvent(
-						'd2l-tab-selected', { bubbles: true, composed: true }
-					));
-				}
+
+		if (changedProperties.has('selected')) {
+			this.ariaSelected = this.selected;
+			if (this.selected) {
```

Comment:
Mostly just restructuring, but this line was modified from checking a string to bool

Reply:
👍 . It looks like the `forEach` was leftover from d2l-tab-internal, where it was checking if either selection or the text changed.